### ISSUE TITLE
[FEAT] Utilize the decrypted private key to sign transactions

### ISSUE THREAD

**Issue by cb7chaitanya** [code100x/tiplink#181]
- Start by enabling the user to actually send some money
- Make the wallet detail component dynamic 
- Goes without saying but you have to get the Solana RPC URL for this to work
- Do tell if you want to use a different RPC provider and give some reasons for it too

**Comment by bishalbera** [comment]
Can I work on this ?

**Comment by cb7chaitanya** [comment]
Go for it @bishalbera

**Comment by basedD3v** [comment]
Mind if I also work on this? I'll create a separate PR, and mods can merge the best one.

**Comment by cb7chaitanya** [comment]
Ask @bishalbera if he is willing to split it, because he is the assignee
And mods isn't the right term, maintainers is @basedD3v

**Comment by basedD3v** [comment]
> Ask @bishalbera if he is willing to split it, because he is the assignee And mods isn't the right term, maintainers is @basedD3v

got it ser, thanks for the correction.

@bishalbera you wanna split the work on this one?

**Comment by cb7chaitanya** [comment]
@basedD3v @bishalbera You have to take a new fork to work on this, please keep your work seperate, take a new fork as commit history has been rewritten

**Comment by bishalbera** [comment]
> > Ask @bishalbera if he is willing to split it, because he is the assignee And mods isn't the right term, maintainers is @basedD3v
> 
> got it ser, thanks for the correction.
> 
> @bishalbera you wanna split the work on this one?

Sorry but  I want to work it out of my own. Thanks

**Comment by bishalbera** [comment]
> @basedD3v @bishalbera You have to take a new fork to work on this, please keep your work seperate, take a new fork as commit history has been rewritten

@cb7chaitanya Thanks for notifying

**Comment by cb7chaitanya** [comment]
Just as a heads up, is this done, because you have to implement only the send Sol functionality? @bishalbera

**Comment by bishalbera** [comment]
@cb7chaitanya no not yet . I'm a beginner that's why it's taking time sorry for that

**Comment by bishalbera** [comment]
@cb7chaitanya update: I've made the sendSol function but do I also need to update the wallet detail component to accept this func cause I dont have AWS credentials.

**Comment by cb7chaitanya** [comment]
How will you test the flow and is the transaction actually happening without getting the cloud credentials? @bishalbera

**Comment by bishalbera** [comment]
> How will you test the flow and is the transaction actually happening without getting the cloud credentials? @bishalbera

Thats true, but I cant use AWS because they wont accept my card thats why I didnt used AWS till now . Also in the readme there wasn't any env var mentioned for AWS (like there is CLIENT_ID and others )thats why I took the issue. Really sorry to hold this issue for a long time. From next time I'll go through the projects more carefully :/ @cb7chaitanya

**Comment by cb7chaitanya** [comment]
In the Readme it isn't yes, but we have AWS & GCP docs listed + id's for it in the env sample too, un assigning you the issue if you can't work on this @bishalbera

**Comment by cb7chaitanya** [comment]
@basedD3v Do you want to work on this?

**Comment by basedD3v** [comment]
Sure!

**Comment by cb7chaitanya** [comment]
Do check out the pre requisites for it, then only pick it up @basedD3v

**Comment by rhithesh** [comment]
I d want in @cb7chaitanya

**Comment by cb7chaitanya** [comment]
Pick it up @rhithesh

**Comment by VineeTagarwaL-code** [comment]
assign me
